ui.py

```diff
@@ -272,28 +306,32 @@ def draw_constraint_listitem(context, layout, constraint):
     )
 
     # Failed hint
-    sub.label(
+    left_sub.label(
         text="",
         icon=("ERROR" if constraint.failed else "CHECKMARK"),
     )
+    # Label
+    left_sub.prop(constraint, "name", text="")
 
     # Middle Part
-    sub = row.row()
-    sub.alignment = "LEFT"
-
-    # Label
-    sub.prop(constraint, "name", text="")
+    center_sub = row.row()
+    center_sub.alignment = "LEFT"
 
     # Dimensional Constraint Values
     for constraint_prop in constraint.props:
-        sub.prop(constraint, constraint_prop, text="")
+        center_sub.prop(constraint, constraint_prop, text="")
+
+    # Disable interaction with element if it is "readonly"
+    center_sub.enabled = (
+        isinstance(constraint, DimensionalConstraint) and constraint.is_reference
```

Comment:
This seems to be inverted, disables constraints that are not only reference and allows editing reference constraints.

Reply:
True, fixed :)